### PR TITLE
Add yautja server builder script

### PR DESCRIPTION
**Pull request by skarred14**

### CODE REVIEW

File: ubuntu20.04/val-verde-platform-sdk-libsecret-cross [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+mkdir -p ${STAGE_ROOT} ${INSTALL_ROOT}
+rsync -aPx ${SOURCE_ROOT}/* \
+          ${STAGE_ROOT}
+
+cd ${STAGE_ROOT}
+${BUILD_PACKAGE_PREFIX}/bin/${VAL_VERDE_GH_TEAM}-platform-sdk-configure \
+    ${STAGE_ROOT}/configure \
+    --disable-static \
+    --disable-gcrypt \
```
Comment (aaditya-chandrasekhar) [inline ubuntu20.04/val-verde-platform-sdk-libsecret-cross:34]: Sort above disable-static.

File: ubuntu20.04/val-verde-platform-sdk-llvm-dependencies-android [hunk 2]
```diff
@@ -120,15 +129,6 @@ else
     bash ${VAL_VERDE_GH_TEAM}-platform-sdk-strace-cross
 fi
 
-# android sed build
```
Comment (aaditya-chandrasekhar) [inline ubuntu20.04/val-verde-platform-sdk-llvm-dependencies-android:123]: why move these?

File: ubuntu20.04/val-verde-platform-sdk-yarn-yautja-server-cross [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+GIT_URL="https://github.com/${PACKAGE_BASE_NAME}/${SOURCE_PACKAGE_NAME}.git" \
+git-clone val-verde-mainline
+
+mkdir -p ${INSTALL_ROOT}${NODE_PATH}/${SOURCE_PACKAGE_NAME}
+
+rsync -aPx ${SOURCE_ROOT}/ \
+           ${STAGE_ROOT}
+cd ${STAGE_ROOT}
+
+AR=${AR:-${BUILD_PACKAGE_PREFIX}/bin/llvm-ar} \
```
Comment (aaditya-chandrasekhar) [inline ubuntu20.04/val-verde-platform-sdk-yarn-yautja-server-cross:29]: The AR and CC defaults can be moved into yarn-build function and applied to the yarn compile step. That way if you want to override AR or CC do so before calling yarn-build.

File: ubuntu20.04/val-verde-platform-sdk-llvm-dependencies-android (hunk 2 above)
Comment (skarred14) [reply ubuntu20.04/val-verde-platform-sdk-llvm-dependencies-android:124]: strace depends on gawk

File: ubuntu20.04/val-verde-platform-sdk-yarn-yautja-server-cross (hunk 3 above)
Comment (skarred14) [reply ubuntu20.04/val-verde-platform-sdk-yarn-yautja-server-cross]: updated

File: ubuntu20.04/val-verde-platform-sdk-libsecret-cross (hunk 1 above)
Comment (skarred14) [reply ubuntu20.04/val-verde-platform-sdk-libsecret-cross:33]: done

**Pull request merged**